owsi-core/owsi-core-components/owsi-core-component-jpa-more/src/main/java/fr/openwide/core/jpa/more/business/property/service/PropertyServiceImpl.java

```diff
@@ -0,0 +1,345 @@
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.TransactionStatus;
+import org.springframework.transaction.interceptor.DefaultTransactionAttribute;
+import org.springframework.transaction.interceptor.TransactionAttribute;
+import org.springframework.transaction.support.TransactionCallback;
+import org.springframework.transaction.support.TransactionCallbackWithoutResult;
+import org.springframework.transaction.support.TransactionTemplate;
+
+import com.google.common.base.Converter;
+import com.google.common.base.Enums;
+import com.google.common.base.Function;
+import com.google.common.base.Optional;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Supplier;
+import com.google.common.collect.Maps;
+import com.google.common.primitives.Doubles;
+import com.google.common.primitives.Floats;
+import com.google.common.primitives.Ints;
+import com.google.common.primitives.Longs;
+
+import fr.openwide.core.commons.util.functional.SerializableSupplier;
+import fr.openwide.core.commons.util.functional.converter.StringBigDecimalConverter;
+import fr.openwide.core.commons.util.functional.converter.StringBooleanConverter;
+import fr.openwide.core.commons.util.functional.converter.StringDateConverter;
+import fr.openwide.core.commons.util.functional.converter.StringDateTimeConverter;
+import fr.openwide.core.commons.util.functional.converter.StringDirectoryFileConverter;
+import fr.openwide.core.commons.util.functional.converter.StringURIConverter;
+import fr.openwide.core.jpa.exception.SecurityServiceException;
+import fr.openwide.core.jpa.exception.ServiceException;
+import fr.openwide.core.jpa.more.business.parameter.model.Parameter;
+import fr.openwide.core.jpa.more.business.parameter.service.IAbstractParameterService;
+import fr.openwide.core.jpa.more.business.property.dao.IImmutablePropertyDao;
+import fr.openwide.core.jpa.more.business.property.dao.IMutablePropertyDao;
+import fr.openwide.core.jpa.more.business.property.model.ImmutablePropertyId;
+import fr.openwide.core.jpa.more.business.property.model.ImmutablePropertyRegistryKey;
+import fr.openwide.core.jpa.more.business.property.model.MutablePropertyId;
+import fr.openwide.core.jpa.more.business.property.model.PropertyId;
+import fr.openwide.core.jpa.more.business.property.model.PropertyRegistryKey;
+import fr.openwide.core.jpa.more.config.spring.event.PropertyServiceInitEvent;
+import fr.openwide.core.spring.config.CoreConfigurer;
+
+/**
+ * Use this service to retrieve application properties instead of using {@link CoreConfigurer} or {@link IAbstractParameterService}.
+ * It handles both mutable and immutable properties ; immutable properties are retrieved from properties resources files
+ * ({@link IImmutablePropertyDao}) and mutable properties are related to {@link Parameter} ({@link IMutablePropertyDao}).
+ * @see {@link IPropertyRegistry} to register application properties.
+ */
+@Service("propertyService")
```

Comment:
You should remove this kind of annotations if you declare beans explicitely with JavaConfig. This is only useful when performing component scanning (which we don't, in this case).